tasks/bin/run_external_test

```diff
@@ -32,6 +32,7 @@ Dir.mktmpdir("chef-external-test") do |dir|
   Dir.chdir(dir) do
     shell_out!("git checkout #{git_thing}", live_stream: STDOUT)
     Bundler.with_unbundled_env do
+      shell_out!("sed -i '/chef-bin/d' Gemfile", live_stream: STDOUT, env: env, timeout: 3600)
```

Comment:
@lamont-granquist No idea if this is a positive move in the right direction to cleanup testing in this repo or a step back. Would you please take time to evaluate and let me know? Tests are now green!

Reply:
I also opened this up to test if the berkshelf pipeline passes without it as well, https://github.com/berkshelf/berkshelf/pull/1869